submissions/dummy_submission/the-nuclear-options-v1.0.md

```diff
@@ -0,0 +1,233 @@
+* **A nuclear reactor is too expensive to maintain without the Kovarex enrichment process** – Indeed, obtaining one unit of Uranium-235 requires centrifuging thousands of uranium ores, but on the other hand 1 piece of it is enough to make 10 uranium fuel cells, which last over 30 minutes in total. In a rather eye-opening [video](https://youtu.be/Qw_NzPuccxk?t=1610) Nilaus walks us through the mathematics and from there we see that you need only 8-9 miners and 1 centrifuge to maintain 1 nuclear reactor continuously without the Kovarex enrichment process, although it does not hurt to process a little more in case of bad luck during centrifuging.
+
+* **Nuclear calculations and designs are hard **– At the end of the day, calculating ratios for nuclear setups is not that different from planning your assembly lines. In the same spirit, the wonderful [Factorio Cheat Sheet](https://factoriocheatsheet.com/#nuclear-power) provides the formulas and interactive calculators you need and there are just a few design rules to keep in mind, which we cover below.
+* **Nuclear plants are bad for the game performance (UPS)** – It’s true that with all the fluid mechanics involved, nuclear cannot compete with the simplicity of solar power in terms of UPS efficiency, but that does not mean its impact is that large, especially with just a handful of reactors.  We will cover this too, below.
+* **What if the radiation causes mutated mega-biters?** - Oddly enough radiation does not seem to affect any entities in vanilla Factorio while regular old pollution already is enough to mutate those pesky biters. Nuclear energy is actually very helpful in this regard, as creates no pollution apart from the relatively small amounts from uranium mining and processing.
+
+With these concerns addressed, let us look into the DIY of nuclear power.
+
+## Designing your own plants
+
+### How do I know how many components I need?
+
+When starting out, it makes sense to start with single reactor, or perhaps a 1x2 (1 row of 2 reactors) layout. The [wiki tutorial](https://wiki.factorio.com/Tutorial:Nuclear_power) has you covered for the basics. After that, the size of your design is based on your power needs. You can arrange your reactors however you want, and the easy way to calculate the power output from a layout is making a single reactor equivalent (SRE) diagram. It works in three steps. We can go over the steps with an example of a 3x3 setup:
+
+`1) Draw a copy of your reactor arrangement and to each reactor assign a value of 1:`
+
+``[1][1][1]` 
+
+``[1][1][1]` 
+
+``[1][1][1]` 
+
+`2) For each reactor, count the number of neighbors it has (they must be aligned exactly side by side) and add it to the 1:`
+
+``[1+2][1+3][1+2] -> [3][4][3]` 
+
+``[1+3][1+4][1+3] -> [4][5][4]` 
+
+``[1+2][1+3][1+2] -> [3][4][3]` 
+
+`3) Sum all the numbers and multiply by 40 to get your power output in MW:`
+
+`(3 + 4 + 3) * 2 + (4 + 5 + 4) = 33 SRE`

```

Comment:
I don't really like using abbreviations unless it's very well known. It's easy to forget what they mean, and then the number looses all meaning. How about something like this maybe?
```suggestion
`(3 + 4 + 3) * 2 + (4 + 5 + 4) = 33 effective reactors`
```
Or simply write it out:
```suggestion
`(3 + 4 + 3) * 2 + (4 + 5 + 4) = 33 equivalent single reactors`
```